contrib/default.config

```diff
@@ -46,18 +46,27 @@ ST_LINUXBOOT_VARIANT=minimal
 # into the LinuxBoot initramfs.
 ST_SIGNING_ROOT="out/keys/example_keys/root.cert"
 
-# ST_HOST_CONFIG_LOCATION defines the location where the host configuration is
-# stored.
+# ST_HOST_CONFIG_MODE defines where the host configuration is stored.
 #
 # initramfs (default): Initramfs of LinuxBoot.
 #
 # cdrom (iso target only): ISO filesystem
 #
 # legacy (disk target only): STBOOT partition
 #
-# efivar: (U)EFI variable "STHostConfig-f401f2c1-b005-4be0-8cee-f2e5945bcbe7"
+# efivar: (U)EFI variable
+#
+ST_HOST_CONFIG_MODE="initramfs"
+
+# ST_HOST_CONFIG_LOCATION defines the location of the host configuration
```

Comment:
`ST_HOST_CONFIG_LOCATION defines the location of the host configuration inside the stboot installation.`
(to avoid confusion with `ST_HOST_CONFIG_FILE`)

Reply:
ack